frontend/src/components/ui/AppDocumentUpload.vue

```diff
@@ -64,6 +92,23 @@ export default {
       type: Array,
       default: () => [],
     },
+    header: {
+      type: String,
+      required: false,
+      default: undefined
+    },
+    showInfoMessage: {
+      type: Boolean,
+      default: true,
+    },
+    showTableHeader: {
```

Comment:
Is this prop required? It always seems to be paired with documentType. It offers flexibility but at the cost of complexity. I think documentType can drive everything.

Reply:
Removed to be implied from documentType